pkg/internal/mock/presentproof/client.go

```diff
@@ -4,7 +4,7 @@ Copyright SecureKey Technologies Inc. All Rights Reserved.
 SPDX-License-Identifier: Apache-2.0
 */
 
-package mockpresentproof
+package presentproof
```

Comment:
I prefer `mockpresentproof.Client` myself.

Reply:
`mock` is already there in the path `pkg/internal/mock/presentproof`. Tried to align this with [aries mocks](https://github.com/hyperledger/aries-framework-go/tree/master/pkg/mock) to be consistent.

Reply:
@rolsonquadras 

> Tried to align this with aries mocks to be consistent.

But why? There was nothing broken.

Reply:
> @rolsonquadras
> 
> > Tried to align this with aries mocks to be consistent.
> 
> But why? There was nothing broken.

It stutters and better to be consistent with other projects.

Reply:
@rolsonquadras 

> > > Tried to align this with aries mocks to be consistent.
> > 
> > 
> > But why? There was nothing broken.
> 
> It stutters and better to be consistent with other projects.

It avoids import collisions, does not stutter in code, was not broken, was already merged, and was placed there as per your request.

Most importantly, these arbitrary changes may impact PRs already open.

Reply:
> @rolsonquadras
> 
> > > > Tried to align this with aries mocks to be consistent.
> > > 
> > > 
> > > But why? There was nothing broken.
> > 
> > 
> > It stutters and better to be consistent with other projects.
> 
> It avoids import collisions, does not stutter in code, was not broken, was already merged, and was placed there as per your request.
> 
> Most importantly, these arbitrary changes may impact PRs already open.

@llorllale what do you mean by "was placed there as per your request." ?

Reply:
https://github.com/trustbloc/edge-adapter/pull/112#discussion_r444205574

Reply:
Yeah, but I don't see I mentioned anything about package name.

Reply:
Now we have two packages without `mock` and one with `mock` prefix - https://github.com/trustbloc/edge-adapter/tree/master/pkg/internal/mock.

It seems like our individual preferences are blocking the PRs.

@sudeshrshetty @fqutishat @llorllale

Reply:
@rolsonquadras fine